src/pages/RequestCallPage.js

```diff
@@ -184,7 +195,15 @@ class RequestCallPage extends Component {
                                 autoCorrect={false}
                                 value={this.state.phoneNumber}
                                 placeholder="+14158675309"
-                                onChangeText={phoneNumber => this.setState({phoneNumber})}
+                                errorText={this.state.phoneNumberError}
+                                onBlur={() => {
+                                    if (_.isEmpty(this.state.phoneNumber.trim())) {
+                                        this.setState({phoneNumberError: this.props.translate('messages.noPhoneNumber')});
+                                    } else if (!Str.isValidPhone(this.state.phoneNumber)) {
+                                        this.setState({phoneNumberError: this.props.translate('requestCallPage.errorMessageInvalidPhone')});
+                                    }
+                                }}
+                                onChangeText={phoneNumber => this.setState({phoneNumber, phoneNumberError: ''})}
```

Comment:
It looks like clearing the phoneNumberError here is causing the error message to disappear as soon as the user starts typing, so the user can't see the phone number example as they type. Can you set the phoneNumberError to empty inside the onSubmit function instead so the error message remains until they resubmit?

Reply:
Yeah